### PR TITLE
Unify Module summaries

### PR DESCRIPTION
**Pull request by eksperimental**
and a few other corrections

### CODE REVIEW

File: lib/elixir/lib/application.ex [hunk 1]
```diff
@@ -1,6 +1,7 @@
 defmodule Application do
   @moduledoc """
-  A module for working with applications and defining application callbacks.
+  Convenience functions for working with applications and defining
```
Comment (josevalim) [inline lib/elixir/lib/application.ex]: The previous definition was much better than "convenience functions".

**Comment by josevalim** [comment]
Thanks but I don't like the unification, honestly. It makes everything too robotic and it would probably make the summary *harder* to read, because everything would be too similar. It is like requiring all chapters in the book to start with the same sentence.

File: lib/elixir/lib/application.ex (hunk 1 above)
Comment (eksperimental) [inline lib/elixir/lib/application.ex]: I was trying to get rid of the repetititve "this modules does..." and  "a module for this and that".

Comment (josevalim) [inline lib/elixir/lib/application.ex]: But then you ended up adding a repetitive "Convenience functions" :D

Comment (eksperimental) [inline lib/elixir/lib/application.ex]: hahaha. that so true.
how about removing the "module thing" and leaving the rest as it was.
there are also a few changes about split long sentences. like the one in kernel.. stufff that shouldb't be in the summary

Comment (josevalim) [inline lib/elixir/lib/application.ex]: Can you please send a PR for the long ones, those would be great. But let's not change the ones under the proper length please. :)

**Pull request closed without merging**